modules/auth/oauth2/oauth2.go

```diff
@@ -24,7 +24,8 @@ import (
 	"github.com/markbates/goth/providers/google"
 	"github.com/markbates/goth/providers/openidConnect"
 	"github.com/markbates/goth/providers/twitter"
-	"github.com/satori/go.uuid"
+	"github.com/markbates/goth/providers/yandex"
+	uuid "github.com/satori/go.uuid"
 	"xorm.io/xorm"
```

Comment:
```suggestion

	"xorm.io/xorm"
```

Reply:
Was not formated before but would be nice if yoz include this :)

Reply:
@lafriks dont add a space there?

Reply:
@6543 "xorm.io/xorm" is an external library, same as "github.com/satori/go.uuid", so they go grouped together. There are only three groups: golang's, "code/gitea.io/gitea" and the rest.